src/lib/hooks/use-wallet.ts

```diff
@@ -10,7 +10,7 @@ export interface IndexRpcProvider {
 }
 
 type Account = {
-  address: Hex | undefined
+  address: string | undefined
```

Comment:
@janndriessen responded to your note in the other comment, but this is the change that doesn't require the `as` casting (and didn't require changes elsewhere either)